backup_and_restore/index.adoc

```diff
@@ -27,3 +26,39 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using a supported version, 1.7, of link:https://velero.io/[Velero]. See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+OADP backs up and restores PVs by using the cloud provider snapshot API, Container Snapshot Interface (CSI) snapshots, or link:https://restic.net/[Restic].
+
+You can install OADP with xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[default plugins].
+
+.OADP requirements
```

Comment:
I'm a bit confused by this section, probably because I normally expect this kind of information before a procedure. Other OCP components list these sorts of requirements before you _do_ something. 

So I'm left wondering. What are these requirements for exactly?

Reply:
This is the landing page for the entire Application Backup and Restore section of the guide. We have to tell them up front what they need. Otherwise, they will waste time going to the procedure to see that they need S3 object storage or snapshot-enabled cloud storage.

Reply:
Also, the user needs to know that they need to be cluster admins.